tests/backup/backup_restore_basic_test.go

```diff
@@ -1442,3 +1443,225 @@ var _ = Describe("{BackupMultipleNsWithSameLabel}", func() {
+					ctx.ReadinessTimeout = appReadinessTimeout
+					namespace := GetAppNamespace(ctx, taskName)
+					log.InfoD("Scheduled application with namespace [%s]", namespace)
+					bkpNamespaces = append(bkpNamespaces, namespace)
+				}
+			}
+		})
+		Step("Validate new namespaces", func() {
+			log.InfoD("Validating new namespaces")
+			ValidateApplications(contexts)
+		})
+		Step("Apply same namespace labels to new namespaces", func() {
+			log.InfoD("Apply same namespace labels to new namespaces")
+			newNamespaces = bkpNamespaces[len(bkpNamespaces)-numberOfNewNamespaces:]
+			err = AddLabelsToMultipleNamespaces(nsLabelsMap, bkpNamespaces[len(bkpNamespaces)-numberOfNewNamespaces:])
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Applying same namespace labels %v to new namespace %v", nsLabelsMap, newNamespaces))
+		})
+		Step("Verify new application namespaces with same namespace label are included in next schedule backup", func() {
+			log.InfoD("Verifying new applications namespace new application namespaces with same namespace label are included in next schedule backup")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			dash.VerifyFatal(err, nil, "Fetching px-central-admin ctx")
+			scheduleBackupAfterAddingNs, err = GetNextScheduleBackupName(scheduleName, 2, ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying backup success for %s schedule backup", scheduleBackupAfterAddingNs))
+			err = NamespaceLabelBackupSuccessCheck(scheduleBackupAfterAddingNs, ctx, bkpNamespaces, nsLabelString)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying if the labeled namespace [%v] is backed up and checks for labels [%s] applied to backup [%s]", bkpNamespaces, nsLabelString, scheduleBackupAfterAddingNs))
+
+		})
+		Step("Restoring scheduled backups", func() {
+			log.InfoD("Restoring scheduled backups")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			dash.VerifyFatal(err, nil, "Fetching px-central-admin ctx")
+			allScheduleBackupNames, err = Inst().Backup.GetAllScheduleBackupNames(ctx, scheduleName, orgID)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching names of all schedule backups of schedule named [%s]", scheduleName))
+			for _, backupName := range allScheduleBackupNames {
+				restoreName = fmt.Sprintf("%s-%s-%v", restoreNamePrefix, scheduleName, time.Now().Unix())
+				err = CreateRestore(restoreName, backupName, namespaceMapping, destinationClusterName, orgID, ctx, nil)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of restoring scheduled backups - %s", restoreName))
+				restoreNames = append(restoreNames, restoreName)
+			}
+		})
+	})
+	JustAfterEach(func() {
+		defer EndPxBackupTorpedoTest(contexts)
+		ctx, err := backup.GetAdminCtxFromSecret()
+		dash.VerifySafely(err, nil, "Fetching px-central-admin ctx")
+		scheduleUid, err := GetScheduleUID(scheduleName, orgID, ctx)
+		dash.VerifySafely(err, nil, fmt.Sprintf("Fetching uid of schedule named [%s]", scheduleName))
+		time.Sleep(1 * time.Minute)
```

Comment:
why this sleep is needed?

Reply:
added that for debugging purposes removed it